launch

```diff
@@ -141,6 +145,8 @@ if __name__ == '__main__':
                               (', '.join(sim_choices.keys()), sim_default)))
     parser.add_argument('-t', '--test', action='store_true',
                         help='Use current stdin & stdout for simulated UART.')
+    parser.add_argument('--ramdisk',
+                        help='Give initial ramdisk to OVPSim. '),
```

Comment:
Would it be difficult to add support for passing ramdisk to QEMU as well? QEMU seems to support `-initrd file` flag, and [this line of malta-specific QEMU code](https://github.com/qemu/qemu/blob/a1cf5fac2b929ffa2abd1285401f2535ff8c6fea/hw/mips/mips_malta.c#L869) suggests that it also uses `rd_start` and `rd_size`.